pkg/sdk/tags_test.go

```diff
@@ -0,0 +1,286 @@
+package sdk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTagCreate(t *testing.T) {
+	t.Run("create with allowed values", func(t *testing.T) {
+		opts := &createTagOptions{
+			OrReplace: Bool(true),
+			name: AccountObjectIdentifier{
+				name: "tag",
+			},
+			AllowedValues: &AllowedValues{
+				Values: []AllowedValue{
+					{
+						Value: "value1",
+					},
+					{
+						Value: "value2",
+					},
+				},
+			},
+		}
+		actual, err := structToSQL(opts)
```

Comment:
please use `assertOptsValidAndSQLEquals` instead of these 4 lines.

Reply:
fixed